modules/mbedtls/crypto_mbedtls.h

```diff
@@ -125,7 +123,7 @@ class CryptoMbedTLS : public Crypto {
 private:
 	mbedtls_entropy_context entropy;
 	mbedtls_ctr_drbg_context ctr_drbg;
-	static X509CertificateMbedTLS *default_certs;
+	static X509CertificateMbedTLS *default_certs = nullptr;
```

Comment:
mmhh.. this is weird though, It's initialized here:
https://github.com/godotengine/godot/blob/master/modules/mbedtls/crypto_mbedtls.cpp#L294

Is this one of those cases where the compiler creates 2 copies of the variable?

Reply:
Ah oops, I missed it. And my init doesn't way anyway, would have to be `inline static`.

Reply:
I don't see what the issue is then, but this PR wouldn't fix it. The other init cleanups can be done in another PR, they're not important.